src/components/CreatePost/CreatePost.tsx

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+
+import { Kweet as KweetModel } from '../../model/kweet';
+import { Kweet } from '../Kweet';
+import {
+  KwitterButton,
+  KwitterPost,
+  KwitterTextArea,
+  ProfilePhoto
+} from './styled';
+
+export const CreatePost = (): React.ReactElement => {
+  const [post, setPost] = useState<string>('');
+  const [click, setClick] = useState<boolean>(false);
+  const [kweetList, setkweetList] = useState<KweetModel[]>([]);
+
+  const dateValue = (value: string) => new Date(value).valueOf();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const fetchUsers = await axios.get('http://localhost:3001/kweets');
+      const saveUsers = await setkweetList(fetchUsers.data);
+      return saveUsers;
+    };
+    fetchData();
+  }, []);
+
+  const kweets = {
+    content: post,
+    kweetedAt: new Date().toISOString(),
+    likes: 0,
+    author: {
+      displayName: 'User A',
+      kwitterHandle: '@user_A',
+      profileImageUrl: 'https://via.placeholder.com/50'
+    }
+  };
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    setkweetList((prev) => [...prev, kweets]);
+    setClick(true);
+    axios.post('http://localhost:3001/kweets', { kweets });
+    setPost('');
+  }
+
+  return (
+    <>
+      <KwitterPost>
+        <ProfilePhoto src="https://via.placeholder.com/50" />
+        <form onSubmit={handleSubmit}>
+          <KwitterTextArea
+            placeholder="What's happening?"
+            value={post}
+            onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) =>
+              setPost(event.target.value)
+            }
+          />
+          <KwitterButton type="submit">Tweet</KwitterButton>
+        </form>
+      </KwitterPost>
+
+      {click &&
+        kweetList
```

Comment:
The sorting of the kweets should be handled by the server-side. I think you can just re-use the `KweetsList` component?